node/node_syncing.go

```diff
@@ -28,6 +29,11 @@ const (
 	MinConnectedPeers = 5  // minimum number of peers connected to in node syncing
 )
 
+var (
+	// DNS harmony dns server for harmony nodes IP
+	DNS = [4]string{"s0.t.hmny.io", "s1.t.hmny.io", "s2.t.hmny.io", "s3.t.hmny.io"}
```

Comment:
why don't you put it in a constant? and add into some internal config package?  like internal/configs/net?  this configuration can be used by other packages like wallet.

Reply:
sure

Reply:
by the way, the string array cannot be a constant, so I use var